ext/popup.js

```diff
@@ -44,22 +50,44 @@ Workspace.all = function() {
 
 Workspace.open = function(workspace) {
   chrome.storage.local.get(workspace.name, function(bucket) {
-    chrome.windows.create({ url: bucket[workspace.name] });
+    chrome.windows.create(function(newWindow) {
+      let isFirstTabLoad = false;
+
+      bucket[workspace.name].forEach((tabInfo) => {
```

Comment:
I think we need to migrate from the old bucket format to the new one. We used to store a list of URLs and now we're storing a complex object...

I'm not sure if this is a blocker to merge this PR or not @marpo60 ?, but we should definitely have to think about data migrations in the short term.